Examples/Framework/include/ActsExamples/EventData/SimSpacePoint.hpp

```diff
@@ -0,0 +1,39 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/Validation/ProtoTrackClassification.hpp"
+namespace ActsExamples {
+struct SimSpacePoint {
+  // Hit id
+  size_t m_Id;
+  // Global position
+  float m_x;
+  float m_y;
+  float m_z;
+  float m_r;
+  // volume and layer ID
+  Acts::GeometryIdentifier m_geoId;
```

Comment:
What is the point to store the geometry id here? I don't see it used anywhere.

Reply:
I used that before, but we don't need it anymore. removed.